Light mode for DetailsPage

feloy opened the pull request:
### What does this PR do?

I have reused `--pd-table-body-text-sub-secondary` for the image id in the header of the image details page. Either we move this variable out of the `table` theming to place in a more generic one, or we create a new entry.

### Screenshot / video of UI

<img width="1118" alt="details-dark" src="https://github.com/containers/podman-desktop/assets/9973512/5ab05dd1-20b1-4534-a4c5-2b922cbb8af5">

![details-light](https://github.com/containers/podman-desktop/assets/9973512/6a62fb73-de55-47d7-9c0e-eaddaeb6ce0f)


### What issues does this PR fix or reference?

Fixes #6927 

### How to test this PR?

<!-- Please explain steps to verify the functionality,
do not forget to provide unit/component tests -->

- [ ] Tests are covering the bug fix or the new feature

deboer-tim reviewed (approved):
Every change reveals more that needs to be done :), but this one looks good. Thanks, approved.

ekidneyrh commented:
I actually have a mock-up made up for this. I tidied it up so it's readable:

[Details page.pdf](https://github.com/containers/podman-desktop/files/15376031/Details.page.pdf)
![Details page](https://github.com/containers/podman-desktop/assets/87311656/8d1ad1a3-95aa-4da9-b6dd-760ffb4f06ff)

For the content-header-icon, you can ignore the colour value there as it follows its own colour scheme depending on whether it's running etc. 

I know there's those mini buttons there too, but I think they can be defined separately. I was unsure about the code part under the kube header - I'm not sure if there is a default styling to that or if it follows a particular rule?

feloy commented:
> I actually have a mock-up made up for this. I tidied it up so it's readable:
> 
> [Details page.pdf](https://github.com/containers/podman-desktop/files/15376031/Details.page.pdf) ![Details page](https://private-user-images.githubusercontent.com/87311656/332035750-8d1ad1a3-95aa-4da9-b6dd-760ffb4f06ff.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYyNzgwMzEsIm5iZiI6MTcxNjI3NzczMSwicGF0aCI6Ii84NzMxMTY1Ni8zMzIwMzU3NTAtOGQxYWQxYTMtOTVhYS00ZGE5LWI2ZGQtNzYwZmZiNGYwNmZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTIxVDA3NDg1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUzZDE1OTBlZWYwZGZjNTU4MDM5ZDk2YmY3OTBkNmUzNmQyY2M5NGQ5ZDBiYWMxNzRiMzVmZDA1YTI2NjMxYzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uJ5HzdzIJO54Oxc7-ACY1ObBRToHwRQw8rs9VhsrKwM)
> 
> For the content-header-icon, you can ignore the colour value there as it follows its own colour scheme depending on whether it's running etc.
> 
> I know there's those mini buttons there too, but I think they can be defined separately. I was unsure about the code part under the kube header - I'm not sure if there is a default styling to that or if it follows a particular rule?

A few comments:

- The tabs are also visible in the list pages. Even if the components are different (Tabs in details page vs `Button type="tab"` in list page), I think we want to have only one set of colors for them. Do we want to keep the `details-tab-text/details-tab-highlight` colors defined with the `details-` prefix, or do we want to move it to a specific prefix, for example `tab-text/tab-highlight`?
- in the mockup, the text color  of the selected tab is the same as the others. Currently, the selected tab has a `white` text color, making it highlighted. Do we want to remove this highlight of the text (and only highlight the border)? If not, I have added a new color `tab-text-highlight`,
- I have added a color (`details-tab-hover`), for the color of the bottom border when a tab is hovered.

ekidneyrh commented:
> The tabs are also visible in the list pages. Even if the components are different (Tabs in details page vs `Button type="tab"` in list page), I think we want to have only one set of colors for them. Do we want to keep the `details-tab-text/details-tab-highlight` colors defined with the `details-` prefix, or do we want to move it to a specific prefix, for example `tab-text/tab-highlight`? 

You're right, we should just have the one set of colours for the tabs/highlight. Since it's reused, I'd go with dropping the details for it. 


> in the mockup, the text color  of the selected tab is the same as the others. Currently, the selected tab has a `white` text color, making it highlighted. Do we want to remove this highlight of the text (and only highlight the border)? If not, I have added a new color `tab-text-highlight`,
> 

Good spot, I'd have that as charcoal-300 in the light mode for the tab-text-highlight.

> I have added a color (`details-tab-hover`), for the color of the bottom border when a tab is hovered.

Would this be similar to the first point that it would be reused, so maybe not have the details- prefix?
I'd suggest having these values be dark mode purple-400, light mode purple-500, same as the input hover :)
